runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
```

Comment:
Yes, what those fds point to depends on the TTY setting, but the main point being that these are open for the application process.